.changeset/clever-hounds-watch.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't think this requires a changeset since it's just a docs change. @markacianfrani whaddya think

Reply:
I forgot he's at a conference all day. Let's just nix this changeset (you can just delete the .md file) since it doesn't actually effect any styling or component functionality

Reply:
@Micah, ok all set!